### ISSUE TITLE
Can't import any module by path

### ISSUE THREAD

**Issue by ChenZerong** [wallabyjs/quokka#5]
### Issue description or question
Maybe I'm configuring something wrong
###  Package.json
``` json
  "engines": {
    "node": ">= 4.0.0",
    "npm": ">= 3.0.0"
  },
  "quokka": {
      "babel":  true
  }
```
### .babelrc
```json
{
  "presets": [
    "es2015"
  , "stage-2"],
  "plugins": ["transform-runtime"],
  "comments": false
}

```
### Quokka.js file

```javascript
import LocalStorage from 'store'
import value from './value.js'
import store from '../vuex/store.js'
.....
```

### Quokka.js Console Output

```
​​​​Quokka #1 (node: v6.9.5, babel: v6.23.1)​​​​
​​​​
​​Cannot find module './value.js'​​
  at ​​​Function.Module._resolveFilename​​​ ​module.js:469​
  at ​​​Function.Module._load​​​ ​module.js:417​
  at ​quokka.js:2:0​
```

### Code editor version

Visual Studio Code v1.10.2

### OS name and version

Linux centos7

**Comment by ArtemGovorov** [comment]
What is the output for this: 
```javascript
console.log(require('fs').existsSync(require('path').join(__dirname, 'value.js')))
```

**Comment by ChenZerong** [comment]
true

**Comment by ArtemGovorov** [comment]
Created this sample with your configs, but can't reproduce the issue: https://github.com/ArtemGovorov/quokka-5. Please clone the repo, and run this quokka:

```javascript
import value from './value.js'

console.log(value)
```

Here's what I'm getting:
![screen shot 2017-03-16 at 12 55 26 pm](https://cloud.githubusercontent.com/assets/979966/23979867/e6f08b94-0a47-11e7-9765-0fd68c22bb52.png)

**Comment by littleStudent** [comment]
I have the same issue with a typescript project.
`Cannot find module "../<path>"`

The example you posted does indeed work as expected.

**Comment by ArtemGovorov** [comment]
@littleStudent Please try creating a reproducible example, happy to have a look.

**Comment by littleStudent** [comment]
not sure if you want me to add this here? or create a new issue? if thats the case just tell me.
https://github.com/littleStudent/quokka-5

**Comment by ArtemGovorov** [comment]
@littleStudent Your sample is [requiring .js file](https://github.com/littleStudent/quokka-5/blob/master/run.ts#L1), but you have `.ts` file. If you remove the extension or change it to `.ts`, and add some basic `tsconfig.json`, then it's working.

**Comment by littleStudent** [comment]
damn, right. bad example.
will try to make a correct reproduction of the issue. I am using it correctly in my project.

**Comment by littleStudent** [comment]
I made some changes. Seems like its not working if you have a deeper folder structure?

**Comment by ArtemGovorov** [comment]
Thanks, I see the issue now. Currently quokka always runs in the context of the project root. The workaround for now is to create an empty quokka (without a real file) and import the `run` file into it.

**Comment by littleStudent** [comment]
alright, so `import * as value from './src/value/value'` would work for quokka instead of `import * as value from '../value/value'`

**Comment by ChenZerong** [comment]
alright, so we now have a way to circumvent this problem, or come to upgrade

**Comment by ArtemGovorov** [comment]
The issue is fixed in v1.0.7, the version is published.

**Comment by littleStudent** [comment]
I can confirm, works for me now.
thx for the fast fix 👍
